.travis.yml

```diff
@@ -1,7 +1,149 @@
 sudo: false
-install: git clone --depth 1 https://github.com/sstephenson/bats.git
-script: make test
 language: c
+
+# virtualenv versions:
+# precise: 1.7.1.2
+# trusty: 1.11.4
+# precise: 12.0.6
+# trusty: 13.1.0
+
+addons:
+  apt:
+    # Note that it is not possible to include `virtualenv` from
+    # source `debian-sid`, as pulls in many other debian-sid packages
+    # which conflicts with the Ubuntu packages.
+    packages:
+      - python-pip
+
 env:
-  - PYENV_NATIVE_EXT=
-  - PYENV_NATIVE_EXT=1
+  global:
+    - VIRTUAL_ENV_BASE=~/virtualenv
+    - VIRTUAL_ENV_VERSION_SPEC='<13'
+
+matrix:
+  include:
+    # Plain bats
+
+    - env: PYENV_NATIVE_EXT=
+      addons: false
+    - env: PYENV_NATIVE_EXT=1
+      addons: false
+
+    # CPython
+
+    - env: TESTENV=2.7.11,3.2.6,3.4.4
+      dist: precise
+
+    # Stackless
+
+    - env: TESTENV=stackless-3.4.1,stackless-dev
+      dist: precise
+
+    # PyPy
+
+    - env: TESTENV=pypy-2.4.0,pypy-5.1.1,pypy-c-jit-latest,pypy-c-nojit-latest
+      dist: precise
+
+    # Jython
+
+    - env: TESTENV=jython-2.7.0,jython-dev VIRTUAL_ENV_VERSION_SPEC='==1.7.1.2'
+      dist: precise
+
+    # Anaconda/Miniconda
+
+    - env: TESTENV=anaconda3-4.0.0,miniconda3-4.0.5 VIRTUAL_ENV_ADD_LIB_PATH=1
+      dist: precise
+
+# Failures (must all be replicated into the allow_failures section)
+
+    # Miniconda 2
+
+    - env: TESTENV=miniconda2-4.0.5
```

Comment:
this is also fixed with `VIRTUAL_ENV_ADD_LIB_PATH=1`
now just need to get ironpython working... ;-)